Taylor/UI/UIView+Coordinates.swift

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2016, Mirego
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+// - Redistributions of source code must retain the above copyright notice,
+//   this list of conditions and the following disclaimer.
+// - Redistributions in binary form must reproduce the above copyright notice,
+//   this list of conditions and the following disclaimer in the documentation
+//   and/or other materials provided with the distribution.
+// - Neither the name of the Mirego nor the names of its contributors may
+//   be used to endorse or promote products derived from this software without
+//   specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+
+import UIKit
+
+/**
+ * Simplified syntax for reading/writing UIView coordinates:
+ - width: CGFloat     [read-write]
+ - height: CGFloat    [read-write]
+ - size: CGSize       [read-write]
+ - origin: CGPoint    [read-write]
+ - x: CGFloat         [read-write]
+ - y: CGFloat         [read-write]
+ - maxX: CGFloat      [read]
+ - maxY: CGFloat      [read]
+ - minX: CGFloat      [read]
+ - minY: CGFloat      [read]
+ - midX: CGFloat      [read]
+ - midY: CGFloat      [read]
+
+ Examples:
+ subview.size = view.size
+ subview.width = view.width
+ */
+// swiftlint:disable variable_name
+extension UIView {
+    var width: CGFloat {
+        set {
+            bounds.size.width = newValue
+        }
+
+        get {
+            return bounds.width
+        }
+    }
+
+    var height: CGFloat {
+        set {
+            bounds.size.height = newValue
```

Comment:
Personally, I prefer always modifying the `frame` property and never the `bounds` property. Don't think it makes a difference but as a convention.

Reply:
It does sadly, `frame` on `UIView` is a computed property that takes into account it's `CALayer` `bounds` and `position` properties and it's own `transform` property. So changing the height with the frame on a view with a transform would cause bad side effect.

Reply:
I would have assumed to have the side effect even with changing the bounds property - good to know!